cmd/kubeadm/app/phases/bootstraptoken/node/tlsbootstrap.go

```diff
@@ -35,9 +34,9 @@ const (
 	// NodeKubeletBootstrap defines the name of the ClusterRoleBinding that lets kubelets post CSRs
 	NodeKubeletBootstrap = "kubeadm:kubelet-bootstrap"
 
-	// CSRAutoApprovalClusterRoleName defines the name of the auto-bootstrapped ClusterRole for making the csrapprover controller auto-approve the CSR
-	// TODO: This value should be defined in an other, generic authz package instead of here
```

Comment:
Leave these two lines as-is. The same nodeclient ClusterRole should still be used.
selfnodeclient is for when the user asks for a renewal. In this case it's the Bootstrap Token that asks for a new credential for a node.